public/mix-manifest.json

```diff
@@ -3,5 +3,6 @@
     "/css/app.css": "/css/app.css",
     "/images/download-stats.svg": "/images/download-stats.svg",
     "/images/user.svg": "/images/user.svg",
-    "/images/wikidata-logo.svg": "/images/wikidata-logo.svg"
+    "/images/wikidata-logo.svg": "/images/wikidata-logo.svg",
```

Comment:
I think we can delete this line, since we deleted the file, right?

Reply:
Indeed, done.